CHANGELOG

```diff
@@ -1,3 +1,6 @@
+1.0.4 - 2025-06-30
```

Comment:
We should release this on coming Monday and later update the composer package in AbTesting/redirect folder and release a new version for that on 7th July